snuba/datasets/outcomes.py

```diff
@@ -64,6 +65,7 @@ def __init__(self):
 
         super(OutcomesDataset, self).__init__(
             schema=schema,
+            ddl=DDL([schema]),
```

Comment:
Once we have read and write schemas, this will look something like this:
```
ddl=DDL([read_schema, write_schema, materialized_view])
```

Reply:
Is "materialized_view" a schema on its own ?